src/video/windows/SDL_windowsopengl.c

```diff
@@ -549,11 +549,11 @@ static int WIN_GL_ChoosePixelFormatARB(_THIS, int *iAttribs, float *fAttribs)
             _this->gl_data->wglChoosePixelFormatARB(hdc, iAttribs, fAttribs,
                                                     1, &pixel_format,
                                                     &matching);
-        }
 
-        /* Check whether we actually got an SRGB capable buffer */
-        _this->gl_data->wglGetPixelFormatAttribivARB(hdc, pixel_format, 0, 1, &qAttrib, &srgb);
-        _this->gl_config.framebuffer_srgb_capable = srgb;
+            /* Check whether we actually got an SRGB capable buffer */
+            _this->gl_data->wglGetPixelFormatAttribivARB(hdc, pixel_format, 0, 1, &qAttrib, &srgb);
+            _this->gl_config.framebuffer_srgb_capable = srgb;
```

Comment:
This second line can (may be should?) be kept out of the if block at its original place? Or, is `_this->gl_config.framebuffer_srgb_capable` initialized to 0 already?

Reply:
I am not sure, I didn't investigated this. Possibly it was initialized already, and in ORIGINAL code there was no of such assignment at all.

Reply:
Yes, it's initialized to NULL.